### PR TITLE
CI: FiX for CI, issue with docker id collision

### PR DESCRIPTION
**Pull request by B-a-S**
## What
Solving collision issue with a place of docker id file and issue with running sshd

## Why ?
CI fail

### CODE REVIEW

File: .ci/job_matrix.yaml [hunk 1]
```diff
@@ -67,8 +67,8 @@ steps:
       echo "INFO: Start docker containers"
       DOCKER_IMAGE_NAME="${UCC_DOCKER_IMAGE_NAME}:${BUILD_ID}"
       docker pull ${DOCKER_IMAGE_NAME}
-      docker create -ti --rm $DOCKER_OPT ${DOCKER_IMAGE_NAME} /bin/bash > ${WORKSPACE}/ucc_docker.id
-      docker start $(cat ${WORKSPACE}/ucc_docker.id)
+      docker create -ti --rm $DOCKER_OPT ${DOCKER_IMAGE_NAME} /bin/bash > /tmp/ucc_docker-"${BUILD_TAG}".id
```
[the remark this answers is not in the document]
Comment (B-a-S) [reply .ci/job_matrix.yaml:70]: It's Jenkins env variable

File: .ci/scripts/run_docker.sh [hunk 2]
```diff
@@ -79,7 +79,7 @@ for HOST in $(cat "$HOSTFILE"); do
     ssh "$HOST" "docker run \
         ${DOCKER_RUN_ARGS} \
         ${DOCKER_IMAGE_NAME} \
-        bash -c 'sudo /usr/sbin/sshd -p ${DOCKER_SSH_PORT}; sleep infinity'"
+        sudo /usr/sbin/sshd -D -p 12345"
```
[the remark this answers is not in the document]
Comment (B-a-S) [reply .ci/scripts/run_docker.sh:82]: Where do you see a secret? ))

**Review by celermajer — APPROVED** [review]
That is very good !! 
Now if sshd failed the container stops and sleep infinity is not called 👍

**Review by amaslenn — APPROVED** [review]

**Comment by artemry-nv** [comment]
Make sure CI passes before we merge this PR.

**Review by artemry-nv — APPROVED** [review]

**Pull request merged**